Tests/fixtures/smartphone-39.yml

```diff
@@ -4709,3 +4709,867 @@
+    engine: Blink
+    engine_version: 99.0.4844.88
+  device:
+    type: smartphone
+    brand: Huawei
+    model: Pura 70
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; arm_64; Android 12; HBN-LX9) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.6261.101 YaBrowser/24.4.4.101.00 SA/3 Mobile Safari/537.36
+  os:
+    name: Android
+    version: "12"
+    platform: ARM
+  client:
+    type: browser
+    name: Yandex Browser
+    version: 24.4.4.101.00
+    engine: Blink
+    engine_version: 122.0.6261.101
+  device:
+    type: smartphone
+    brand: Huawei
+    model: Pura 70 Pro+
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 5.1; Gravity 5.5 LTE) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.110 Mobile Safari/537.36
+  os:
+    name: Android
+    version: "5.1"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 70.0.3538.110
+    engine: Blink
+    engine_version: 70.0.3538.110
+  device:
+    type: smartphone
+    brand: Maxwest
+    model: Gravity 5.5 LTE
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 14; V2352; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/87.0.4280.141 Mobile Safari/537.36 VivoBrowser/11.2.1.0
+  os:
+    name: Android
+    version: "14"
+    platform: ""
+  client:
+    type: browser
+    name: vivo Browser
+    version: 11.2.1.0
+    engine: Blink
+    engine_version: 87.0.4280.141
+  device:
+    type: smartphone
+    brand: Vivo
+    model: Y28
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; arm_64; Android 14; RMX3910) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.6261.39 YaBrowser/24.4.7.39.00 SA/3 Mobile Safari/537.36
+  os:
+    name: Android
+    version: "14"
+    platform: ARM
+  client:
+    type: browser
+    name: Yandex Browser
+    version: 24.4.7.39.00
+    engine: Blink
+    engine_version: 122.0.6261.39
+  device:
+    type: smartphone
+    brand: Realme
+    model: C65
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10; Y51 PK) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Mobile Safari/537.36
+  os:
+    name: Android
+    version: "10"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 87.0.4280.88
+    engine: Blink
+    engine_version: 87.0.4280.88
+  device:
+    type: smartphone
+    brand: Vivo
+    model: Y51 PK
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 5.1; Fun +) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.99 Mobile Safari/537.36
+  os:
+    name: Android
+    version: "5.1"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 71.0.3578.99
+    engine: Blink
+    engine_version: 71.0.3578.99
+  device:
+    type: smartphone
+    brand: Plusone
```

Comment:
Are we sure it's the brand? Nothin in user_agent indicates this. and it would make more sense if it's https://www.imei.info/phonedatabase/orbic-funplus/ if it's a phone even or https://apkpure.net/fun-plus-xc/com.nnetwork.funplus

Reply:
orbic fun+  android OS 12 version

https://www.gsmchoice.com/en/catalogue/orbic/funplus4g/

Reply:
which means the brand is incorrect, no?

Reply:
brand, Freetel was founded by Plus One Marketing in 2012
https://ja.wikipedia.org/wiki/FREETEL

I returned the name back, also deleted PlusOne